modules/nf-core/propr/propr/tests/main.nf.test

```diff
@@ -0,0 +1,158 @@
+nextflow_process {
+
+    name "Test Process PROPR_PROPR"
+    script "../main.nf"
+    process "PROPR_PROPR"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "propr"
+    tag "propr/propr"
+
+    test("Test propr/propr using default options") {
+
+        tag "default"
+
+        when {
+            process {
+                """
+                input[0] = [
+                    [ id:'test' ],
+                    file(params.test_data['mus_musculus']['genome']['rnaseq_matrix'], checkIfExists: true)
+                ]
+                """
+            }
+        }
+        
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(process.out.matrix).match("Test propr/propr using default options") },
```

Comment:
Why only snapshotting the matrix?

Reply:
The FDR file is calculated based on a random sampling, so I don´t think we can snapshot.
Then the rds is the R object including the matrix, and the fdr, and more stuff that could be useful to run propr again if the user wants. So, same problem cannot be snapshot and have the same md5

Reply:
Can you use set.seed() to make the FDR reproducible? You're right on the rds of course

Reply:
That would require to change the package code and rebuild container and so on...
It is really a test that shuffles the input data and run multiple times the function used to create the matrix output, so I think if the matrix output is consistent, this step should not have problem too

Reply:
@pinin4fjords now it should be fine. I added a way to fix the seed for fdr reproducibility